backend/models/Question.js

```diff
@@ -30,15 +31,16 @@ const questionSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  category:{
+  category: {
     type: String,
-    required: true
+    required: true,
+    enum: categoryKeys
   },
-  commentId: {
+  comments: {
```

Comment:
Renamed to `comments` so that it can be populated using mongoose `populate` method